File: BE-FindingJob/src/main/java/com/example/befindingjob/dto/CompanyDTO.java
package com.example.befindingjob.dto;

import lombok.*;

@Data
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class CompanyDTO {
    private int companyId;
    private String companyName;
    private String description;
    private String address;
    private String logoUrl;
    private String website;
    private int userId;
}

File: JobHub/app/src/main/java/com/example/jobhub/dto/CompanyDTO.kt
package com.example.jobhub.dto

import com.google.gson.annotations.SerializedName

data class CompanyDTO(
    @SerializedName("companyId") var companyId: Int? = null,
    @SerializedName("companyName") var companyName: String? = null,
    @SerializedName("description") var description: String? = null,
    @SerializedName("address") var address: String? = null,
    @SerializedName("logoUrl") var logoUrl: String? = null,
    @SerializedName("website") var website: String? = null,
    @SerializedName("userId") var userId: Int? = null
)
